Modify SmartFund token info

fergus789 opened the pull request:
**.json and .png document checklists**
- [X] logo dimensions: 120*120 px
- [X] logo background: Transparent
- [X] At least one transaction on the token contract
- [X] For ERC-20 project teams: Upper and lower capitalized alphabets in token address (For checksum purposes)
- [x] Same document name for both .json and .png (Including capitalization)

**Project background:**

Official announcement:#imToken #4653 We are providing SF token’s information on imToken. After completion, you can see the logo and full information inside imToken’s Token Manager.

Project team background:The SmartFund team is located at Singapore. The core members have more than 10 years of experience in Internet product development, and they are also a technical team in the field of early research blockchain. At present, there are more than a dozen members of the team, which gathers the core technology development elites of the wallets, BPs, mining, trading, etc., and has experience in operating LBTC Block Producer.

Project basic information:The SmartFund is a set of smart contracts and tokens running on Ethereum. The smart investment ecosystem consists of a $mart Fund and a number of DAPP projects invested, the core goal is to use the smart contract to achieve a digital currency decentralized investment in the DAPP projects with clear profit model. The democratic vote participates in the investment decision and automatically distributes the project revenue in real time.

Tradeable exchanges:https://coinx.exchange

Recommended Gas Limit (For ERC-20 project teams) for transaction: 5000000

fergus789 commented:
@tstt  I modified the symbols and icons of the token, but they are not updated in the Imtoken. Could you please check it out? Thank you~

tstt commented:
> @tstt I modified the symbols and icons of the token, but they are not updated in the Imtoken. Could you please check it out? Thank you~

Sorry, we've fixed the update issues, you can pull request again. We'll check with it asap.

fergus789 commented:
> > @tstt I modified the symbols and icons of the token, but they are not updated in the Imtoken. Could you please check it out? Thank you~
> 
> Sorry, we've fixed the update issues, you can pull request again. We'll check with it asap.

It doesn't matter, please check #4707 .
